test/relations.test.js

```diff
@@ -2527,42 +2527,6 @@ describe('relations', function() {
         done();
       });
     });
-
-    it('should allow to create belongsTo model in beforeCreate hook', function(done) {
-      var mind;
-      Fear.beforeCreate = function(next) {
-        this.mind.create(function(err, m) {
-          mind = m;
-          if (err) next(err); else next();
-        });
-      };
-      Fear.create(function(err, fear) {
-        should.not.exists(err);
-        should.exists(fear);
-        fear.mindId.should.be.equal(mind.id);
-        should.exists(fear.mind());
-        done();
-      });
-    });
-
-    it('should allow to create belongsTo model in beforeCreate hook with promises', function(done) {
```

Comment:
Yeah, I think it was not necessary to create a promise-based version of the test above, it's good this is going away.